.github/workflows/ci.yml

```diff
@@ -114,7 +114,11 @@ jobs:
       run: |
         # do not use fakeroot, but run as root. avoids the dreaded EISDIR sporadic failures. see #2482.
         #sudo -E bash -c "tox -e py"
-        tox --skip-missing-interpreters
+        tox --skip-missing-interpreters -- --junitxml=test-results.xml
+    - name: Upload test results to Codecov
+      uses: codecov/test-results-action@v1
+      with:
+        files: test-results.xml
```

Comment:
thanks for the PR!

the docs you linked talk about using !cancelled() here to make sure it gets executed even if tests fail.

Reply:
ping @katia-sentry - did you see my feedback?